.gitignore

```diff
@@ -12,3 +12,6 @@ project/boot/
 project/plugins/project/
 
 .idea*
+
+# Nexus credentials configuration
+nexus.sbt
```

Comment:
I'd rather have the credentials in a hidden subdirectory of the home directory, e.g. `~/.sbt/.credentials_sml_nexus` (note also a dedicated file name, since many people have other credentials configured as well, so just using `.credentials` can cause conflicts.

Reply:
done